src/main/java/de/azapps/kafkabackup/common/partition/PartitionException.java

```diff
@@ -0,0 +1,10 @@
+package de.azapps.kafkabackup.common.partition;
+
+
+public class PartitionException extends Exception {
```

Comment:
RuntimeException instead?

Reply:
You mean replacing the PartitionException with RuntimeException, or inherit from RuntimeException?

Reply:
inherit from RuntimeException